scripts/voucherHubUtils.ts

```diff
@@ -81,7 +86,9 @@ export async function getExpectedVoucherProxyCodeHash(voucherBeaconAddress: stri
          *
          * Note: Look very carefully before updating this value to avoid messing with
          * existing vouchers already deployed in production.
+         *
+         * Also see test/NextVersionUpgrade.test.ts to double check behavior.
          */
-        return '0x1891638e9af48f5f31af8ab5c97eaccc41dcb99bf8bdd8909759f674e4a0a9a6';
+        return '0x31a4f4707138270dd34b040129096c67e1039fb242deebb8a0d0f8ed9da82232';
```

Comment:
Why does the proxy code hash changes?

Reply:
The hash depends on the Voucher beacon address. The Voucher beacon address depends on the address of the first Voucher implementation, which is updated in this PR (deployment from scratch)